src/App.tsx

```diff
@@ -11,14 +11,26 @@ import Kandidatdetaljer from './pages/kandidat/Kandidatdetaljer';
 import Oversikt from './pages/oversikt/Oversikt';
 import Registrering from './pages/registrering/Registrering';
 import './styles/app.less';
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
 
 const App: FunctionComponent = () => (
     <div className="app">
         <Normaltekst tag="div">
             <FilterContextProvider>
                 <Router basename={APP_ROOT}>
-                    <Header />
+                    {process.env.ON_HEROKU && (
+                        <AlertStripeFeil>
+                            <b>Dette er kun en testversjon.</b>
+                            <br />
+                            Skal du registrere eller søke etter kandidater? Åpne denne lenken i
+                            Chrome SKSS. <br />
+                            <a href="https://arbeidsgiver.nais.adeo.no/finn-kandidat/">
```

Comment:
Skulle vi ikke _ikke_ ha dette som lenke?

Reply:
Kan også vurdere å ta dette inn i en komponent? Det er jo tross alt "test"-kode, burde ikke forsøple for mye av det i main-kode

Reply:
Har ikke fått committa det enda, du er for rask

Reply:
>:D